### PR TITLE
Add dependency on moveit_resources_panda_moveit_config. 

### LINKED ISSUE #59: Migrate From `panda_moveit_config` to `moveit_resources_panda_moveit_config`
### Description

As discussed here: https://github.com/ros-planning/moveit2_tutorials/blob/main/package.xml#L40-L43

Blocks a bunch of other tutorials.

### PR DESCRIPTION
**Pull request by DLu**
### Description

Fixes #59.

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

**Review by JafarAbdi — APPROVED** [review]

File: package.xml [hunk 1]
```diff
@@ -64,7 +59,6 @@
   <run_depend>tf2_geometry_msgs</run_depend>
   <run_depend>xacro</run_depend>
 
-  <test_depend>moveit_resources_panda_moveit_config</test_depend>
```
Comment (MarqRazz) [inline package.xml:67]: I thought this was needed for the [tests](https://github.com/ros-planning/moveit2_tutorials/tree/main/doc/tests) tutorial?? We can remove it if that's not the case.

Comment (DLu) [reply package.xml:67]: I thought it would be superseded by the `run_depend`

**Comment by rhaschke** [comment]
@ros-planning/moveit-maintainers: I suggest reverting these changes. 
The tutorials should not depend on `moveit_resources_panda_moveit_config` (which is a minimal version of `panda_moveit_config` referring to a minimal version of the URDF model as well), but use the official `panda_moveit_config` package instead, which uses the official URDF from `franka_description`.

**Comment by 130s** [comment]
> @ros-planning/moveit-maintainers: I suggest reverting these changes. The tutorials should not depend on `moveit_resources_panda_moveit_config` (which is a minimal version of `panda_moveit_config` referring to a minimal version of the URDF model as well), but use the official `panda_moveit_config` package instead, which uses the official URDF from `franka_description`.

Ticketed https://github.com/ros-planning/moveit2_tutorials/issues/704 for clarification.

**Pull request merged**